.rubocop-https---raw-githubusercontent-com-facebook-chef-cookbooks-main--rubocop-55-yml

```diff
@@ -0,0 +1,220 @@
+AllCops:
```

Comment:
please don't commit this file. The linters should download it automatically

Reply:
Ya, I'm just a dumbass. Fixed in [2918e24](https://github.com/facebook/grocery-delivery/pull/64/commits/2918e24fe501766edb2c40beddd7b5bc991eedb9).